docs/05-technical-reference/00-custom-resources/evnt-01-subscription.md

```diff
@@ -43,21 +39,11 @@ This table lists all the possible parameters of a given resource together with t
 |-------------|:---------:|--------------|
 | **metadata.name** | Yes | Specifies the name of the CR. |
 | **metadata.namespace** | No | Defines the Namespace in which the CR is available. It is set to `default` unless your specify otherwise. |
-| **spec.filter** | Yes | Defines the list of filters. |
-| **spec.filter.dialect** | No | Specifies the preferred Eventing backend. Currently, the capability to switch between Eventing backends is not available. It is set to NATS by default. |
-| **spec.filter.filters** | Yes | Defines the filter element as a combination of two CloudEvent filter elements. |
-| **spec.filter.filters.eventSource** | Yes | The origin from which events are published. |
-| **spec.filter.filters.eventType** | Yes | The type of events used to trigger workloads. |
-| **spec.filter.filters.eventSource.property** | Yes | Must be set to `source`. |
-| **spec.filter.filters.eventSource.type** | No | Must be set to `exact`. |
-| **spec.filter.filters.eventSource.value** | Yes | Must be set to `""` for the NATS backend. |
-| **spec.filter.filters.eventType.property** | Yes | Must be set to `type`. |
-| **spec.filter.filters.eventType.type** | No | Must be set to `exact`. |
-| **spec.filter.filters.eventType.value** | Yes | Name of the event being subscribed to, for example: `sap.kyma.custom.commerce.order.created.v1`. The name cannot contain any non-alphanumeric characters `[^a-zA-Z0-9]+`. Read [Event names](../evnt-01-event-names.md#event-name-cleanup) for more information. |
-| **spec.protocol** | Yes | Must be set to `""`. |
-| **spec.protocolsettings** | Yes | Defines the CloudEvent protocol setting specification implementation. Must be set to `{}`. |
-| **spec.sink** | Yes | Specifies the HTTP endpoint where matching events should be sent to, for example: `test.test.svc.cluster.local`.  |
-| **spec.config.maxInFlightMessages** | No | The maximum idle "in-flight messages" sent by NATS to the sink without waiting for a response. By default, it is set to 10.  |
+| **spec.typeMatching** | No | Defines the matching type (`standard` or `exact`) for event types. When it is set to `exact`, Eventing will not do any kind of modifications to the provided `spec.types` internally. In case of `standard`, Eventing will modify the types internally to fulfil the backend requirements. It is set to `standard` unless you specify otherwise. |
```

Comment:
```suggestion
| **spec.typeMatching** | No | Defines the matching type (`standard` or `exact`) for event types. When it is set to `exact`, Eventing does not do any kind of modifications to the provided `spec.types` internally. In case of `standard`, Eventing  modifies the types internally to fulfil the backend requirements. It is set to `standard` unless you specify otherwise. |
```